packages/vkui/src/components/HorizontalCell/HorizontalCell.tsx

```diff
@@ -78,10 +93,13 @@ export const HorizontalCell = ({
   getRootRef,
   getRef,
   extraSubtitle,
+  textAlign: textAlignProp,
+  noPadding = false,
   ...restProps
 }: HorizontalCellProps): React.ReactNode => {
   const hasTypography =
     hasReactNode(title) || hasReactNode(subtitle) || hasReactNode(extraSubtitle);
+  const textAlign = textAlignProp ?? (size === 's' ? 'center' : 'start');
```

Comment:
Можно сделать так, чтобы не создавать промежуточную переменную `textAlignProp`
```suggestion
  textAlign = size === 's' ? 'center' : 'start',
  noPadding = false,
  ...restProps
}: HorizontalCellProps): React.ReactNode => {
  const hasTypography =
    hasReactNode(title) || hasReactNode(subtitle) || hasReactNode(extraSubtitle);
```